tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -64,9 +64,12 @@ RUN apt-get update --allow-insecure-repositories && \
     rm -rf /var/lib/apt/lists/*
 
 
-# Override the MIOpen install with a newer version that has fusion API fixes/updates
+# Override the default rocBLAS and MIOpen installs with a newer version with support for BnA fusion
+RUN mkdir -p $HOME/pkgs/rocblas
+RUN cd $HOME/pkgs/rocblas && wget https://www.dropbox.com/s/dvjmyoeyoiotc5s/rocblas-0.14.3.0-Linux.deb && dpkg -i *.deb
```

Comment:
for 0.14.3.0, can we use official release package on github?

https://github.com/ROCmSoftwarePlatform/rocBLAS/releases

Reply:
Yes, that'll also work.

Reply:
Yes that is better...will make the change once CI completes